### PR TITLE
Added locale: zh_CN

### PR DESCRIPTION
**Pull request by Senorsen**

### CODE REVIEW

File: src/Locales/zh_CN.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        "h"      => '1小时',
+        "hh"     => '%d小时',
+        "d"      => '1天',
+        "dd"     => '%d天',
+        "M"      => '1个月',
+        "MM"     => '%d个月',
+        "y"      => '1年',
+        "yy"     => '%d年',
+    ),
+    "ordinal"       => function ($number, $period)
```
Comment (fightbulc) [inline src/Locales/zh_CN.php]: Hey guys, where do you take the ```$period``` from?

File: src/Locales/zh_CN.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+// author: Senorsen : https://github.com/Senorsen
+
+return array(
+    "months"        => explode('_', '一月_二月_三月_四月_五月_六月_七月_八月_九月_十月_十一月_十二月'),
+    "monthsShort"   => explode('_', '1月_2月_3月_4月_5月_6月_7月_8月_9月_10月_11月_12月'),
+    "weekdays"      => explode('_', '星期日_星期一_星期二_星期三_星期四_星期五_星期六'),
+    "weekdaysShort" => explode('_', '周日_周一_周二_周三_周四_周五_周六'),
+    "weekdaysMin"   => explode('_', '日_一_二_三_四_五_六'),
+    "calendar"      => array(
+        "sameDay"  => function() {
```
Comment (fightbulc) [inline src/Locales/zh_CN.php]: function gets passed a ```Moment``` object. So you should not use ```$this``` but actually do the following:

```php
// e.g. for "sameDay"
"sameDay"  => function(Moment\Moment $moment) {
    return $moment->getMinute() === 0 ? '[今天]Ah[点整]' : '[今天]LT';
}
```

**Comment by fightbulc** [comment]
Guys, thanks for your support. However, please check the remarks. I will move it out of the master for this moment. Looking forward to your next pull request.

**Comment by fightbulc** [comment]
I had a quick look. Maybe you misunderstood what ordinal numbers are? I found this by doing a quick search: http://en.wikipedia.org/wiki/Chinese_numerals#Ordinal_numbers

That looks different to what you guys tried. Can you have a look again. Thanks.

**Comment by Senorsen** [comment]
Thanks for pointing out these & sorry for my mistakes and rudeness pull request! I'll check more carefully.

**Comment by fightbulc** [comment]
Hey there. Just saw that you took it from the Moment.js locale. Our APIs differ in some ways. I see that you might need the ```token``` to determine which symbol to use for the correct ordinal. I'll have a look and let you know.

**Comment by fightbulc** [comment]
@Senorsen have a look I did some changes and pushed it up again: https://github.com/fightbulc/moment.php/blob/master/src/Locales/zh_CN.php

I think you need to adjust the tokens within the calendar, e.g. ```Ah``` and ```LT```. These are still momentjs tokens.

Update your fork and adjust these things, test it and send me a new pull request.

**Pull request merged**